lib/tbd/psi.rb

```diff
@@ -1678,8 +1678,8 @@ def process(model = nil, argh = {})
     # around edge with respect to a reference vector (perpendicular to the
     # edge), +clockwise as one is looking in the opposite position of the edge
     # vector. For instance, a vertical edge has a reference vector pointing
-    # North - surfaces eastward of the edge are (0°,180°], while surfaces
-    # westward of the edge are (180°,360°].
+    # North - surfaces eastward of the edge are (0deg,180deg], while surfaces
+    # westward of the edge are (180deg,360deg].
```

Comment:
Unexpectedly, the _TBD Tests_ suite generates US-ASCII "\xC2" (°) encoding errors when running GitHub Actions. This seems limited to JSON files, and for Ruby 3 (i.e. SDK v3.8.0 & v3.9.0). Weird - as the very same files are processed when running _TBD_ GitHub Actions, which did not yield any encoding errors. Something of a Hail Mary to get rid of the annoyance.

Not particularly interested in keeping the ° char (instead of "deg", for instance), so either deleting or substituting all instances, in both JSON files and Ruby file comments.

Reply:
Weird that characters in comments caused an issue, but this must trigger some file encoding, possibly when checked out by GitHub?

Reply:
Good point. I suspect that the only culprit is the _tbd.schema.json_ file. Just decided to purge/replace ° in ALL files, regardless.